docs/markdown.rst

```diff
@@ -2,56 +2,98 @@ markdown
 ========
 
 .. dfhack-tool::
-    :summary: Exports the text you see on the screen for posting online.
-    :tags: unavailable
-
-This tool saves a copy of a text screen, formatted in markdown, for posting to
-Reddit (among other places). See `forum-dwarves` if you want to export BBCode
-for posting to the Bay 12 forums.
-
-This script will attempt to read the current screen, and if it is a text
-viewscreen (such as the dwarf 'thoughts' screen or an item 'description') then
-append a marked-down version of this text to the output file. Previous entries
-in the file are not overwritten, so you may use the ``markdown`` command
-multiple times to create a single document containing the text from multiple
-screens, like thoughts from several dwarves or descriptions from multiple
-artifacts.
-
-The screens which have been tested and known to function properly with this
-script are:
-
-#. dwarf/unit 'thoughts' screen
-#. item/art 'description' screen
-#. individual 'historical item/figure' screens
-#. manual pages
-#. announcements screen
-#. combat reports screen
-#. latest news (when meeting with liaison)
-
-There may be other screens to which the script applies. It should be safe to
-attempt running the script with any screen active. An error message will inform
-you when the selected screen is not appropriate for this script.
+    :summary: Save description of selected unit/item in Markdown file
+    :tags: dfhack | items | units
+
+Saves the description of a selected unit or item to a Markdown file encoded in UTF-8.
+
+By default, data is stored in the `Dwarf Fortress/markdown_{YourWorldName}.md`
+file in the root game directory.
+
+For units, the script retrieves:
+#. Name, race, age, profession
+#. Description from the Unit/Health/Description screen
+#. Traits from the Unit/Personality/Traits screen
+The script tested on dwarves, dogs, elves, goblins, beasts,
+and is expected to work with all units
+
+For items, it retrieves:
+#. Decorated name (e.g., "☼«☼Chalk Statue of Dakas☼»☼")
+#. Full description from the item's view sheet
+The script works for most items with in-game descriptions and names, including those in storage,
+on the ground, installed as a building, or worn/carried by units.
+
+By default, entries are appended, not overwritten, allowing the ``markdown`` command
+to compile descriptions of multiple items & units in a single document.
```

Comment:
it might also be useful to explain how to create a keybinding for output to a custom filename, or maybe how to use `gui/quickcmd` to manage the workflow. E.g. if "markdown -o description" is the first entry in `gui/quickcmd`, then `Ctrl-Shift-a, a` would run that command.

Reply:
I'd rather add it to the next release, as I have never used (custom) global hotkeys and do not feel comfortable adding info about them now.

Reply:
can be added in a separate PR, absolutely. it should be in the same DFHack *release*, though, since this script won't be nearly as useful without a global hotkey.

Reply:
After testing it now, after long break, I see that the hotkey is an absolute must-have here
But I need this sweet drug of my code being accepted for the first time 🙏